static/css/webflow.css

```diff
@@ -483,6 +486,9 @@ textarea.w-select {
 .w-col-offset-11 {
   margin-left: 91.66666667%;
 }
+.lang-col {
+  min-height: 200px;
+}
```

Comment:
this should not be in this file. this will be overwritten whenever we do a update from webflow. 
option 1: create the style also in webflow
option 2: put it in the rtd.css file
option 3: create a customize.css file.

option 3 might be the best one. option 2 the one to go.